### PR TITLE
[Feature]: Performance Settings IDs and Audio Links

### PR DESCRIPTION
**Pull request by thepenguinco**
**Requires:**
- Schema upgrade

**Description:**
- Include performance settings IDs with performances,
- Include audio and performance links with performances
- Update the corresponding APIs

**Testing Notes:**
- Regression testing on the existing performance APIs

### CODE REVIEW

File: pages/api/performances/create.js [hunk 1]
```diff
@@ -13,20 +13,23 @@ export default async (req, res) => {
...
+    competitionLevelID,
     eventID,
     schoolID,
   } = req.body;
 
   // If required fields were not provided, return an error
-  if (!danceEntry || !eventID || !schoolID) {
+  if (!eventID || !schoolID) {
     return res.status(400).json({
       error: 'Required fields danceEntry, eventId, or schoolID were not provided',
```
Comment (feng-eric) [inline pages/api/performances/create.js:34]: Remove danceEntry from the error message

File: pages/api/performances/edit.js [hunk 2]
```diff
@@ -14,29 +14,40 @@ export default async (req, res) => {
...
+    danceSizeID,
+    danceStyleID,
+    competitionLevelID,
     eventID,
     schoolID,
   } = req.body;
 
+  // TODO: patch edit per data field?
+  /** Optional FIELDS
+    danceSizeID,
```
Comment (feng-eric) [inline pages/api/performances/edit.js:35]: If you don't pass them in, does it update it to NULL?

**Review by feng-eric — APPROVED** [review]
Other than the 2 comments I had, it looks good to me

File: pages/api/performances/edit.js (hunk 2 above)
Comment (thepenguinco) [reply pages/api/performances/edit.js:34]: Aye let me try this out.

**Review by feng-eric — APPROVED** [review]

**Pull request merged**